python/paddle/v2/framework/tests/op_test_util.py

```diff
@@ -21,19 +21,15 @@ def __new__(cls, name, bases, attrs):
         obj = super(OpTestMeta, cls).__new__(cls, name, bases, attrs)
 
         def test_all(self):
-            func = getattr(creation.op_creations, self.type, None)
-            self.assertIsNotNone(func)
-
             scope = core.Scope()
             kwargs = dict()
-            places = []
-            places.append(core.CPUPlace())
+            places = [core.CPUPlace()]
             if core.is_compile_gpu():
                 places.append(core.GPUPlace(0))
 
             for place in places:
-                for in_name in func.all_input_args:
-                    if hasattr(self, "inputs") and in_name in self.inputs:
+                for in_name in Operator.get_op_input_names(self.type):
```

Comment:
If we have a `OpWrapper` class, we can use
```python
for in_name in Operator.Info("type").inputs():
   pass
```
and remove [get_op_input_names ](https://github.com/PaddlePaddle/Paddle/pull/3237/files#diff-e05d4a53c32f65d0b3099f8af73749e1R198)